src/screens/ShowDetailsScreen.tsx

```diff
@@ -61,16 +56,7 @@ export default function ShowDetailsScreen(): JSX.Element {
                     <div>
                         <p className='max-w-md'>{details.overview}</p>
                     </div>
-                    {providers?.results.US !== undefined ?
-                        <div>
-                            {/* #161 TODO: Provide service logo instead of string with styling and positioning */}
-                            {providers?.results.US.flatrate.map((item, i) => (
-                                <span data-testid='details-provider' key={i}>{item.provider_name} </span>
-                            ))}
-                        </div>
-                        :
-                        <p>Sorry, no providers available for this show.</p>
-                    }
+                    <Providers ID={details.id} />
```

Comment:
nit: 
```suggestion
                    <Providers id={details.id} />
```
Generally speaking, file or application constants should be all caps. Since this value is variable I think it makes more sense to be lowercase. Despite the fact it is an acronym.